### PR TITLE
[#11103] Audit logs: right-align timestamps

### LINKED ISSUE #11103: Audit logs: right-align timestamps
Perhaps easier on the eye if we right align timestamps, so that hours/mins/seconds line up?
image

EDIT: It looks like the time stamp has a fixed width (i.e., the same number of characters) but the hours/seconds don't line up because we are not using a mono-spaced font for the timestamp.
Compare
* Mon, 12 Apr
* Fri, 09 Apr

with
* `Mon, 12 Apr`
* `Fri, 09 Apr`

See how the second one lines up nicely.
Perhaps the better solution is to use a mono-spaced font for the timestamp (rather than right-align)?

### PR DESCRIPTION
**Pull request by ypinhsuan**
Fixes #11103

Before
logs_cut

After
updated logs

### CODE REVIEW

**Comment by damithc** [comment]
@ypinhsuan After seeing the result of right-aligning (which looks OK but not great), I took another look at the original issue and added an alternative idea. What do you think about it?

**Comment by ypinhsuan** [comment]
I think using mono-spaced font looks better than right align. I will try changing the font to mono-spaced

**Comment by ypinhsuan** [comment]
This is how it looks like with monospace font
monospace

**Comment by damithc** [comment]
> This is how it looks like with monospace font
> monospace

I'm OK with this option. While it is not ideal to use two different fonts in the table, I think we can live with it.

File: src/web/app/components/sortable-table/sortable-table.component.ts [hunk 1]
```diff
@@ -111,4 +112,12 @@ export class SortableTableComponent implements OnInit {
...
     this.sortRows();
   }
+
+  getFont(cellData: SortableTableCellData): String {
+    if (cellData.font === undefined) {
+      return '';
+    }
+
+    return cellData.font;
+  }
```
Comment (moziliar) [inline src/web/app/components/sortable-table/sortable-table.component.ts:122]: Can ask if this UI will expect different fonts from different data set?

Comment (ypinhsuan) [reply src/web/app/components/sortable-table/sortable-table.component.ts:122]: Only timestamps will be using a different font. The `cellData .font` for other data sets will be undefined and they will use the font in `styles.scss`

File: src/web/app/components/sortable-table/sortable-table.component.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ export interface ColumnData {
 export interface SortableTableCellData {
   value?: any; // Optional value used for sorting with sortBy provided in ColumnData
   displayValue?: string; // Raw string to be display in the cell
+  font?: String;
```
Comment (ccyccyccy) [inline src/web/app/components/sortable-table/sortable-table.component.ts:24]: Use typescript's `string` type instead of the javascript's `String` type.

**Review by ccyccyccy — CHANGES_REQUESTED** [review]
Just a minor issue. Other than that, LGTM!

**Review by ccyccyccy — APPROVED** [review]

**Review by rrtheonlyone — APPROVED** [review]

**Pull request merged**